syntax = "proto3";

import "adlinktech/datariver/descriptor.proto";

package com.adlinktech.example.protobuf;


message Observation {
  option(.adlinktech.datariver.tag_group) = {
    qosProfile: "telemetry"
    description: "ADLINK Edge SDK Example Camera observation TagGroup"
  };

  string barcode = 1 [(.adlinktech.datariver.field_options) = {
    description: "Barcode"
    unit: "n/a"
  }];
  int32 position_x = 2 [(.adlinktech.datariver.field_options) = {
    description: "x coordinate of position"
    unit: "n/a"
  }];
  int32 position_y = 3 [(.adlinktech.datariver.field_options) = {
    description: "y coordinate of position"
    unit: "n/a"
  }];
  int32 position_z = 4 [(.adlinktech.datariver.field_options) = {
    description: "z coordinate of position"
    unit: "n/a"
  }];
};
